torch/csrc/api/include/torch/nn/modules/any.h

```diff
@@ -431,7 +431,11 @@ AnyModule::AnyModule(std::shared_ptr<ModuleType> module)
       torch::detail::has_forward<ModuleType>::value,
       "Can only store module with a forward() method that has a non-templatized"
       "return type into AnyModule (e.g. we cannot store nn::Sequential"
```

Comment:
Would you like to mention `.. a non-templatized argument type and return type into AnyModule` here? It could help with surfacing the issue when someone else tried to do the same thing as we did :)